django_project/user_map/templates/user_map/mail/old_users_email.html

```diff
@@ -0,0 +1,8 @@
+Dear InaSAFE Users, Developers, Trainers
+
+We are excited to announce the launch of our new user map (http://users.inasafe.org). The bad news is that you will have to add yourself to the new map if you wish to continue to be seen by your InaSAFE friends. We hope that you will add yourself again.
+
+On the new map, you can add multiple roles of InaSAFE and roles of OpenStreetMap. If you are a certified InaSAFE trainer or a certified OSM trainer, please send us an e-mail to info@inasafe.org so that you have the cool badges!
```

Comment:
@timlinux @mach0 this is the email that we are going to send to our users from the old user map. If you want to use another email, you can just change this file before we send it